decidim-core/app/assets/javascripts/decidim/input_mentions.js.es6

```diff
@@ -0,0 +1,47 @@
+// = require typeahead.bundle
+
+$(() => {
+  const $mentionContainer = $('.js-mentions-container');
+
+  const sources = [
```

Comment:
Why do we have this static data here? Isn't this used only as an example? If so, it should live only inside the design app (under `app/assets`).

Reply:
No, no, it's yet in progress. That's an example, you may ignore it. I'll provide further info when release. 😀 
Nevertheless, `input_tags.js` is ready. You could check this out. I've tried to pay attention in how you add vendor & javascripts to the app, because they're gonna straight to the core.

Reply:
Oh, sorry! I was in a review spree and didn't see this was actually in progress. Ignore me! And good job!